### DISCUSSION TITLE
Combined image version in quay.io

### DISCUSSION THREAD

**Discussion by jfkxs** [argoproj/argo-events#1249] [Q&A] [answered]
Hello,

Is there a way to know what the combined image version is in quay.io? Maybe I'm not looking at the right place, but it seems like there is only a "latest" tag there quay.io/argoproj/argo-events:latest
The only version I could find in the image was the argo cli version, but I'm more interested in the argo-events version. The "not combined" images in dockerhub are published with version tags, but I guess these will be deprecated in favor of the combined image?

Great project. We love it.
Thanks!

**Comment by whynowy** [comment, marked as answer]
Image combination happens after v1.3, so currently only `latest` tag exists quay.io.
